Core/Config.cpp

```diff
@@ -680,7 +680,7 @@ static const ConfigSetting controlSettings[] = {
 	ConfigSetting("AnalogIsCircular", &g_Config.bAnalogIsCircular, false, CfgFlag::PER_GAME),
 	ConfigSetting("AnalogAutoRotSpeed", &g_Config.fAnalogAutoRotSpeed, 8.0f, CfgFlag::PER_GAME),
 
-	ConfigSetting("AnalogLimiterDeadzone", &g_Config.fAnalogLimiterDeadzone, 0.6f, CfgFlag::PER_GAME),
+	ConfigSetting("AnalogLimiterDeadzone", &g_Config.fAnalogLimiterDeadzone, 0.25f, CfgFlag::DEFAULT),
```

Comment:
This used to be only used when the "analog limiter" button was pressed.  It seems like it has completely different meaning now?  Something related to disabling the analog speed setting?

"Analog limiter" basically suppressed the volume (while pressed) of the analog stick movement (VIRTKEY_ANALOG_LIGHTLY.)  I've rarely used it, from searching the code I feel like it probably doesn't do anything anymore.

-[Unknown]

Reply:
Oh! Looks like I indeed confused things with each other.

I thought this was the analog deadzone for pressing the analog speed button, couldn't figure out what else it might have been. It was hooked up to precisely nothing.

But then it surely should be named some kind of Fast-forward audio limiter, instead, and not contain the word deadzone?

Reply:
I think I'll just get rid of it instead.